brands/shop/furniture.json

```
@@ -170,6 +170,18 @@
       "shop": "furniture"
     }
   },
+  "shop/furniture|  "Courts Malaysia": {
```

Whoa looks like an extra quote here is messing things up.

```suggestion
  "shop/furniture|Courts Malaysia": {
```